### DISCUSSION TITLE
File Manager with thumb preview in roadmap?

### DISCUSSION THREAD

**Discussion by SandroRiz** [Megabit/Blazorise#2397] [Ideas]
Just curious if in the roadmap there is a File manager with files&folders tree navigation and maybe with thumb preview of pictures.
Something like [Syncfusion commercial component](https://www.syncfusion.com/blazor-components/blazor-file-manager)

Thanks

**Comment by stsrki** [comment]
It all depends on how my [plan](https://blazorise.com/news/changes-to-blazorise-license/) will work. If the commercial licensing would be successful and I manage to get some help then yes, definitely will implement a similar component. And even more of them.

**Comment by SandroRiz** [comment]
I completely agree with your decision (and my company is > 1M , but if the amount is reasonable and the level of work is pretty good like yours, there is no-excuse for buy a license).

Hope Megabit will be a MegaSuccess !!

**Reply by stsrki** [reply]
I sincerely hope so too :)

BTW. The new commercial web is already online and is ready, but I'm waiting with sharing the link until my lawyer prepares [SLA](https://en.wikipedia.org/wiki/Service-level_agreement). Hope it will be done during next week.
